legate/driver/launcher.py

```diff
@@ -50,9 +50,6 @@
     "NVIDIA_",
 )
 
-# this will be replaced by bind.sh with the actual computed rank at runtime
-LEGATE_GLOBAL_RANK_SUBSTITUTION = "%%LEGATE_GLOBAL_RANK%%"
-
 
 class Launcher:
```

Comment:
Also remove "rank_id" from docstring and class member type annotations.

Reply:
https://github.com/nv-legate/legate.core/pull/471/commits/256ee056e3b15421ec12994629388ba0e7161cce